source/includes/courses/_users.md

```diff
@@ -0,0 +1,233 @@
+# Users
+
+## Get all users in a course
+
+```shell
+curl https://cerego.com/api/v3/courses/:id/users
+    -H "Content-Type: application/json"
+    -H "Authorization: Bearer <BEARER_TOKEN>"
+```
+
+> The above command returns JSON structured like this:
+
+```json
+{
+  "data": [
+    {
+      "id": "1158898",
```

Comment:
TIL Kyle... is Abe Lincoln...

Reply:
reincarnated on his bday

Reply:
Did anyone read the example JSON responses on the last PR?

Reply:
Oh boy, nope, am now, gonna have to sign up for Genghis Khan's World Domination course